File: RaedBundle/Entity/Reclamation.php
<?php

namespace SlimBundle\Entity;

/**
 * Reclamation
 */
class Reclamation
{
    /**
     * @var string
     */
    private $nomproduit;

    /**
     * @var string
     */
    private $sujet;

    /**
     * @var string
     */
    private $contenu;

    /**
     * @var integer
     */
    private $etat;

    /**
     * @var integer
     */
    private $idreclamation;

    /**
     * @var \AppBundle\Entity\Utilisateur
     */
    private $idutilisateur;


    /**
     * Set nomproduit
     *
     * @param string $nomproduit
     *
     * @return Reclamation
     */
    public function setNomproduit($nomproduit)
    {
        $this->nomproduit = $nomproduit;

        return $this;
    }

    /**
     * Get nomproduit
     *
     * @return string
     */
    public function getNomproduit()
    {
        return $this->nomproduit;
    }

    /**
     * Set sujet
     *
     * @param string $sujet
     *
     * @return Reclamation
     */
    public function setSujet($sujet)
    {
        $this->sujet = $sujet;

        return $this;
    }

    /**
     * Get sujet
     *
     * @return string
     */
    public function getSujet()
    {
        return $this->sujet;
    }

    /**
     * Set contenu
     *
     * @param string $contenu
     *
     * @return Reclamation
     */
    public function setContenu($contenu)
    {
        $this->contenu = $contenu;

        return $this;
    }

    /**
     * Get contenu
     *
     * @return string
     */
    public function getContenu()
    {
        return $this->contenu;
    }

    /**
     * Set etat
     *
     * @param integer $etat
     *
     * @return Reclamation
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;

        return $this;
    }

    /**
     * Get etat
     *
     * @return integer
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * Get idreclamation
     *
     * @return integer
     */
    public function getIdreclamation()
    {
        return $this->idreclamation;
    }

    /**
     * Set idutilisateur
     *
     * @param \AppBundle\Entity\Utilisateur $idutilisateur
     *
     * @return Reclamation
     */
    public function setIdutilisateur(\AppBundle\Entity\Utilisateur $idutilisateur = null)
    {
        $this->idutilisateur = $idutilisateur;

        return $this;
    }

    /**
     * Get idutilisateur
     *
     * @return \AppBundle\Entity\Utilisateur
     */
    public function getIdutilisateur()
    {
        return $this->idutilisateur;
    }
}

File: SAV/src/entites/Reclamation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entites;

/**
 *
 * @author pc hp
 */
public class Reclamation {
     private int idreclamation;
   public static int idutilisateur=12;
      public  String nomproduit;
    public String sujet;
    public String contenu;
    public int etat=0;

    public Reclamation(String nomproduit, String sujet, String contenu) {
        this.nomproduit = nomproduit;
        this.sujet = sujet;
        this.contenu = contenu;
    }

    public Reclamation() {
    }

    public Reclamation(String sujet, String contenu) {
        this.sujet = sujet;
        this.contenu = contenu;
   
    }
  

    public Reclamation(int idutilisateur, String nomproduit, String sujet, String contenu, int etat) {
        this.idutilisateur = idutilisateur;
           this.nomproduit = nomproduit;
        this.sujet = sujet;
        this.contenu = contenu;
        this.etat = etat;
     
    }

    public Reclamation(int idreclamation, int idutilisateur,String nomproduit, String sujet, String contenu, int etat) {
        this.idreclamation = idreclamation;
        this.idutilisateur = idutilisateur;
        this.nomproduit = nomproduit;
        this.sujet = sujet;
        this.contenu = contenu;
        this.etat = etat;
        
    }

    public int getIdreclamation() {
        return idreclamation;
    }

    public int getIdutilisateur() {
        return idutilisateur;
    }

    public String getSujet() {
        return sujet;
    }

    public String getContenu() {
        return contenu;
    }

    public int getEtat() {
        return etat;
    }

    public String getnomproduit() {
        return nomproduit;
    }
    
    @Override
    public String toString() {
        return   nomproduit;
    }

    public void setIdreclamation(int idreclamation) {
        this.idreclamation = idreclamation;
    }

    public void setIdutilisateur(int idutilisateur) {
        this.idutilisateur = idutilisateur;
    }

    public void setSujet(String sujet) {
        this.sujet = sujet;
    }

    public void setContenu(String contenu) {
        this.contenu = contenu;
    }

    public void setEtat(int etat) {
        this.etat = etat;
    }

    public void setnomproduit(String nomproduit) {
        this.nomproduit = nomproduit;
    }

  
    
    
}
